root/static/scripts/edit/URLCleanup.js

```diff
@@ -1754,6 +1754,15 @@ const CLEANUPS = {
       return {result: false};
     },
   },
+  'irombook': {
+    match: [new RegExp('^(https?://)?staticbrainz\\.org/irombook/', 'i')],
```

Comment:
If there is no need for cleanup, there is probably no need for flexible matching either:

```suggestion
    match: [new RegExp('^https://staticbrainz\\.org/irombook/')],
```